dotnetv3/AutoScaling/scenarios/AutoScale_Basics/AutoScale_Basics/AutoScaleMethods.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AutoScale_Basics
+{
+    public class AutoScaleMethods
+    {
+        // snippet-start:[AutoScale.dotnetv3.AutoScale_Basics.CreateAutoScalingGroup]
+        /// <summary>
+        /// Creates a new Amazon Auto Scaling group.
+        /// </summary>
+        /// <param name="client">The  initialized Amazon Auto Scaling client object.</param>
```

Comment:
The  initialized Amazon Auto Scaling client object. --> The initialized Amazon EC2 Auto Scaling client object.

Reply:
fixed